### PR TITLE
Add option to centralize deauth to management node

### LINKED ISSUE #757: Add option to centralize deauth on management node
Otherwise all servers have to be configured as RFC3576

Easy with some vendors but hard with some

### PR DESCRIPTION
**Pull request by julsemaan**
# Description

Adds an option to centralize deauth to management node of active/active cluster
# Impacts

Deauthentication in active/active cluster
Adds a listen interface of the webservices to the management IP
# Issue

fixes #757
# Delete branch after merge

YES
# NEWS file entries
## New Features
- Added an option to centralize deauthentication on the management node of an active/active cluster

### CODE REVIEW

File: conf/pf.conf.defaults [hunk 1]
```diff
@@ -1093,6 +1093,11 @@ password=1234
 #
 # Shared KEY for vrrp protocol (Must be the same on all members).
 virtual_router_id=50
+#
+# active_active.centralized_deauth
+#
+# Centralize the deauthentication to the management node of the cluster.
+centralized_deauth=disabled
```
Comment (louismunro) [inline conf/pf.conf.defaults]: I think the default should be enabled.
There are no downsides that I can see, whereas the reverse can fail in ways that are hard to debug for the uninitiated.

**Comment by louismunro** [comment]
LGTM. Let's do this.

**Comment by louismunro** [comment]
Please rebase. Also make this option the default.

File: conf/pf.conf.defaults (hunk 1 above)
Comment (julsemaan) [inline conf/pf.conf.defaults]: Good point I'll reverse it

**Comment by julsemaan** [comment]
Should be ready

**Pull request merged**